templates/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- start fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end fonts -->
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
+            <a href="#">My Logo</a>
+          </div>
+
+          <div class="nav__container__parnav__bars">
+            <div class="nav__container__parnav__bars__responsive">
+              <img
+                src="..//static/imgs/Union@2x.png"
+                alt="Union@2x"
+                class="responsive nav__container__parnav__bars__responsive__respo"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <!-- end nav -->
+    <!-- start sectionone -->
+    <section class="sectionone" id="sectionone">
```

Comment:
Kindly also try to fix the background image of this section to match the Figma design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/191700801-f191e934-e64b-41f0-87bf-dd7e54a4d980.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/191700863-96fa8226-754e-4adb-a743-f7d456bda27b.png)

Reply:
- Thanks 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊